content/en/internal_developer_portal/scorecards/_index.md

```diff
@@ -30,9 +30,13 @@ further_reading:
 
 ## Overview
 
-Scorecards help you monitor, prioritize, plan, and communicate effectively to take informed actions that improve your software's health and performance. Each scorecard shows the status for Production Readiness, Observability Best Practices, and Documentation & Ownership. All entities with defined metadata in the Software Catalog are automatically evaluated against a set of pass-fail criteria.
+Scorecards help teams measure and continuously improve the health and performance of their software. As Platform Engineers, you can create Scorecards to automatically evaluate entities in your Software Catalog against defined criteria to surface areas that need attention.
```

Comment:
```suggestion
Scorecards help your team measure and continuously improve the health and performance of your software. As Platform Engineers, you can create Scorecards to automatically evaluate entities in your Software Catalog against defined criteria to surface areas that need attention.
```
I noticed that this paragraph shifted from third person to second person, so I thought this would help things sound a little more consistent.